api/graphite.go

```diff
@@ -0,0 +1,350 @@
+package api
+
+import (
+	"errors"
+	"net/http"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/raintank/dur"
+	"github.com/raintank/metrictank/api/middleware"
+	"github.com/raintank/metrictank/api/models"
+	"github.com/raintank/metrictank/api/rbody"
+	"github.com/raintank/metrictank/cluster"
+	"github.com/raintank/metrictank/consolidation"
+	"github.com/raintank/metrictank/idx"
+	"github.com/raintank/worldping-api/pkg/log"
+	"gopkg.in/raintank/schema.v1"
+)
+
+var MissingOrgHeaderErr = errors.New("orgId not set in headers")
+var MissingQueryErr = errors.New("missing query param")
+var InvalidFormatErr = errors.New("invalid format specified")
+var MaxPointsPerReqErr = errors.New("request would execed maxPointsPerReq setting")
+var MaxDaysPerReqErr = errors.New("request would execed maxDaysPerReq setting")
```

Comment:
exceed typo.
but actually, I prefer the original errors like "too many targets/maxDataPoints requested" and "too many targets/too large timeframe requested", because those errors were more geared towards the user, talking in terms of the request. these new errors refer to settings that are geared towards the operator of the system. it's not like the user (typically) has any control over the settings of the server, but they do have control over their requests. although we could guide them on how to approach the operator, or do it themselves if they run their own MT, by saying something like
"please reduce the amount of targets or maxDataPoints in your request, or ask your operator to increase maxPointsPerReq"